RELEASES.md

```diff
@@ -1,3 +1,144 @@
+Version 1.19.0 (2017-07-20)
+===========================
+
+Language
+--------
+
+- [Numeric fields can now be used for creating tuple structs.][36868]
+  For example `struct Point(u32, u32); let x = Point { 0: 7, 1: 0 };`.
+- [Macro recursion limit increased to 1024 from 64.][41676]
+- [Added lint for detecting unused macros.][41907]
+- [`loop` can now return a value with `break`.][42016]
+  For example: `let x = loop { break 7; };`
+- [C compatible `union`s are now available.][42068] They can only contain `Copy`
+  types and cannot have a `Drop` implementation.
+  Example: `union Foo { bar: u8 }`
+
+Compiler
+--------
+
+- [Added bootstrap support for Android.][41370]
+- [Change `arm-linux-androideabi` to correspond to the `armeabi`
+  official ABI.][41656]
+- [Fixed ICE when removing a source file between compilation sessions.][41873]
+- [Minor optimisation of string operations.][42037]
+- [Compiler error message is now `aborting due to previous error(s)` instead of
+  `aborting due to N previous errors`][42150] This was previously inaccurate and
+  would only count certain kinds of errors.
+- [Now supports Visual Studio 2017][42225]
+- [Added a lot][42264] of [new error codes][42302]
+
+Libraries
+---------
+
+- [`String` now implements `FromIterator<Cow<'a, str>>` and
+  `Extend<Cow<'a, str>>`][41449]
+- [`Vec` now implements `From<&mut [T]>`][41530]
+- [`SplitWhitespace` now implements `Clone`][41659]
+- [`[u8]::reverse` is now 5x faster and `[u16]::reverse` is now
+  1.5x faster][41764]
+
+Compatibility Notes
+-------------------
+
+- [`MutexGuard<T>` may only be `Sync` if `T` is `Sync`.][41624]
+- [`-Z` flags are now no longer allowed to be used on the stable
+  compiler.][41751] This has been a warning for a year previous to this.
+- [Ending a float literal with `._` is now a hard error.
+  Example: `42._` .][41946]
+- [Ending a str literal with an underscore is now a hard error
+  Example: `"foo"_`][41990]
+- [Publicly exposing a private type is now a hard error][34537] This was
+  previously a warning.
+- [Type parameter defaults in trait impls and functions is now a
+  hard error][36887] This was previously a warning.
```

Comment:
This is still a warning, there was too much breakage (as you can see, the issue is still open).